### PR TITLE
Drop repeat

### PR DESCRIPTION
**Pull request by Mubra**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -31,11 +33,6 @@ install:
 - python bootstrap.py
 - bin/buildout $VERSIONS annotate
 - bin/buildout $VERSIONS
-before_script:
-  - export ROBOT_SELENIUM2LIBRARY_RUN_ON_FAILURE="Capture Page Screenshot"
```
Comment (hvelarde) [inline .travis.yml]: don't remove this line.

File: .travis.yml [hunk 2]
```diff
@@ -31,11 +33,6 @@ install:
...
-  - export DISPLAY=:99.0
-  - sh -e /etc/init.d/xvfb start
-  - firefox -v
```
Comment (hvelarde) [inline .travis.yml]: neither this one.

File: .travis.yml (hunk 1 above)
Comment (Mubra) [reply .travis.yml]: @hvelarde ok, remove all of the 'before_script' part because it wasn't in the comment of @idgserpro

**Comment by idgserpro** [comment]
@hvelarde @Mubra 

I think this PR should be closed and created another, just to fix Travis.

**Pull request closed without merging**